internal/clients/team4/judge.go

```diff
@@ -92,6 +92,30 @@ func (j *judge) InspectHistory(iigoHistory []shared.Accountability, turnsAgo int
 
 // GetPardonedIslands decides which islands to pardon i.e. no longer impose sanctions on
 // COMPULSORY: decide which islands, if any, to forgive
+func (j *judge) GetPardonedIslands(currentSanctions map[int][]shared.Sanction) map[int][]bool {
+	pardons := map[int][]bool{}
+
+	internalParams := j.parent.internalParam
+	maxPardonTime := internalParams.maxPardonTime
+	maxTierToPardon := internalParams.maxTierToPardon
+	minTrustToPardon := internalParams.minTrustToPardon
```

Comment:
Why do you create these? Each name only seems to be used once anyway?

Reply:
readablity

Reply:
Hmm... I'd kinda disagree as firstly it's so short it doesn't make a difference, and secondly, it's nice to know when reading further down that these variables are our internal parameters and not global configs for example. But it's fine. I'm not asking you to change it

Reply:
changed ;)